aws/ecs-service/main.tf

```
@@ -79,6 +79,27 @@ resource "aws_ecs_task_definition" "this" {
   tags = local.tags
 }
 
+resource "aws_ecs_task_definition" "job" {
```

Do we want this to be optional? So that we can only run jobs when we need to?

We probably want a `count` here with a conditional variable:

```suggestion
resource "aws_ecs_task_definition" "job" {
  count = var.create_job_task ? 1 : 0
```